sdk/src/interfaces.ts

```diff
@@ -88,20 +93,22 @@ export interface ProjectOptions {
    */
   hideDevTools?: boolean;
   /**
-   * Hide the sidebar on page load.
-   *
-   * Users will still be able to open the sidebar by clicking one of the sidebar icons.
+   * Hide the ActivityBar (sidebar icons).
    */
   hideExplorer?: boolean;
   /**
-   * Hide the preview URL.
+   * Show the sidebar as open or closed on page load.
+   *
+   * This might be ignored on narrow viewport widths (mobile and/or tablets).
+   *
+   * On larger viewports, defaults to `false` for `embedProject*` methods, and `true` for `openProject*` methods.
    */
-  hideNavigation?: boolean;
+  showSidebar?: boolean;
```

Comment:
This one is a bit hard to describe because it signals an intent BUT we may not respect it in some cases.

Especially on small screen widths (<1000px), we would hide the sidebar by default even if receiving `showSidebar=1`. I didn't describe the full implementation (which we're also tweaking right now), but used a comment that suggests we might do something different on small screens.